### ISSUE TITLE
Unique event name

### ISSUE THREAD

**Issue by wanglihe2010** [open-learning-exchange/planet#4796]
It looks the calendar event title has to be unique, even among different enterprises. Is this required?

**Comment by paulbert** [comment]
No, event names should not be unique across enterprises.  I'm not sure about within an enterprise, even, but we should at least only make them unique within the enterprise.
